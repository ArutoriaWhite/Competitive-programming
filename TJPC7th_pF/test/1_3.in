65676662 2
8475406 0
36435232 1
