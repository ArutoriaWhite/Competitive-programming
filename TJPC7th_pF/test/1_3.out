8475406 27959826
